roles/ha_apps/etcd/files/ha-etcd-template.yml

```diff
@@ -0,0 +1,94 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: etcd
+  labels:
+    app: etcd
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: etcd-client
+  namespace: etcd
+  labels:
+    app: etcd
+spec:
+  type: LoadBalancer
+  ports:
+  - name: etcd-client
+    port: 2379
+    protocol: TCP
+    targetPort: 2379
+  selector:
+    app: etcd
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: etcd
+  namespace: etcd
+  labels:
```

Comment:
You might also wanna add annotation in the metadata that looks like this 
```
  annotations:
    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
```
This will make sure that the dns entries of pods are created even when they are bootstrapping. This annotation is depracated in favor of `publishNotReadyAddresses` [here](https://github.com/kubernetes/kubernetes/pull/63742). But I have had issues with `publishNotReadyAddresses`, I suggest using both and removib the annotation when the latter is supported accross all DNS implementations.

Reply:
@alaypatel07  Thanks for review! Is everything else ok, besides annotations?